tests/unifiedPlatform/pds2/pds_dataservice_test.go

```diff
@@ -136,6 +148,8 @@ var _ = Describe("{ScaleUpCpuMemLimitsandStorageOfDS}", func() {
 
 	JustAfterEach(func() {
 		defer EndPDSTorpedoTest()
+		WorkflowDataService.UpdateDeploymentTemplates = false
```

Comment:
Could we handle this in common.go/EndPDSTorpedo test instead of After Each.

Reply:
Done